source/fuzz/fuzzer_util.cpp

```diff
@@ -1352,6 +1361,21 @@ opt::Instruction* GetLastInsertBeforeInstruction(opt::IRContext* ir_context,
   return CanInsertOpcodeBeforeInstruction(opcode, &*it) ? &*it : nullptr;
 }
 
+bool HasBuiltInMembers(opt::IRContext* ir_context, uint32_t struct_type_id) {
+  const auto* type = ir_context->get_type_mgr()->GetType(struct_type_id);
+  (void)type;  // Make compilers happy in release mode.
+  assert(type && type->AsStruct() &&
+         "|struct_type_id| is not a result id of an OpTypeStruct");
```

Comment:
You could replace this with a single assertion checking that the opcode of struct_type_id when looked up in the def use manager is OpTypeStruct.  That would avoid the need for the "keep compilers happy" cast.  I don't feel strongly either way here.

Reply:
Fixed in fdc3b5d.